src/Symfony/Component/OptionsResolver/OptionResolverNested.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\OptionsResolver;
+
+class OptionResolverNested extends \ArrayObject
```

Comment:
should it be named `NestedOption`, any better idea? `OptionResolverNested` looks a bit strange to me